src/test/lib/operators/maintenance/create_table_test.cpp

```diff
@@ -39,6 +39,13 @@ TEST_F(CreateTableTest, NameAndDescription) {
   EXPECT_EQ(create_table->name(), "CreateTable");
   EXPECT_EQ(create_table->description(DescriptionMode::SingleLine),
             "CreateTable 't' ('a' int NOT NULL, 'b' float NULL)");
+
+  const auto context = Hyrise::get().transaction_manager.new_transaction_context(AutoCommit::No);
```

Comment:
??

Reply:
I fixed a minor issue that came up while debugging something. Should have left a note. It was impossible to print the description of a CreateTable operator after its input has been cleared. This is now fixed. To test this, I need to execute it; to execute it, I need a tx context.